src/main/kotlin/org/jitsi/nlj/Transceiver.kt

```diff
@@ -256,6 +256,7 @@ class Transceiver(
 
     // TODO(brian): we may want to handle local and remote ssrc associations differently, as different parts of the
     // code care about one or the other, but currently there is no issue treating them the same.
+    // note(george): we need to be careful with this not to cause any side-effects with rtx
```

Comment:
can you add to this comment? what's the potential problem scenario?

Reply:
i don't think this causes any issues with plain re-transmissions but i haven't gone through an exhaustive search of potential problematic scenarios.

main motivation for this note was that i had to understand why the firefox rtp sender had rtx ssrcs, since we're not supposed to have any with firefox, which led me to this line.

maybe it needs to be reworded or scrap it completely?

Reply:
yeah i was just thinking of rewording it a bit to make sure someone could come in and understand the implication without having to ask you.  as it is is just a bit ambiguous.  when i read this i think: "what about ssrc associations do i need to be careful about so i don't break rtx?"  so i think just some more detail, e.g.: "learning about a remote rtx ssrc association could confuse code to think it has rtx enabled when it actually doesn't" or something?

but, also, maybe it should be scrapped as you seem to be working around it fine?  should we add a field which denotes an ssrc association as relating to a remote or local ssrc to better fix this?

Reply:
i scraped the note, it felt redundant and unnecessarily scary; we can surely think about adding such a field, but maybe at later time? for now the code seems to work fine